mycpp/mycpp_main.py

```diff
@@ -370,7 +371,12 @@ def main(argv):
     log('\tmycpp pass: DATAFLOW')
     stack_roots = None
     if opts.minimize_stack_roots:
-        stack_roots = pass_state.ComputeMinimalStackRoots(cfgs)
+        souffle_dir = tempfile.TemporaryDirectory()
```

Comment:
I think it could be useful to view the temp dir when debugging, rather than automatically cleaning it up

Not sure if that's feasible though

In any case, It would be nice to have a comment showing what the directory layout is

Reply:
I can add an environment var (an option would be cumbersome since it would require adding an extra translator) to let you specify the output dir when debugging

Reply:
Will also add a comment about the directory layout

Reply:
Added `MYCPP_SOUFFLE_DIR`. Let me know what you think